### ISSUE TITLE
improve accessibility of the `<graph>` component

### ISSUE THREAD

**Issue by virginiaMae** [Doenet/DoenetML#652]
Currently, there is no label that alerts the user that they are entering a graph region. If the description attribute is included, this is read off first. Then, the screen reader starts reading off the features of the graph. This includes the x-axis, the x-axis labels, followed by the y-axis, the y-axis labels, followed by features on the graph, followed by the navigation. All of the graphical features are simply read off as either "graphic symbol" or the number (such as an axis label or point coordinate), and there is no information that describes what the feature that is being read off refers to on the graph. Based on a comparison with similar graphical tools on Desmos Amplify classroom activities involving graphs, I recommend:

- Before the alt-text is read off, and before any features are read, the user should first hear "Doenet Interactive Graph. You are now on a Graph."

- Next, the alt-text should be read.

- Following this, there should be an option for the user to provide an interaction to enter the graph. (On VoiceOver for example this is Cntl-Option-Shift-Down Arrow)

- Once the user enters the graph, we need to apply labels to all features of the graph as they are read off. Some of these features should be suppressed if the feature cannot generally be interacted with. For example, instead of reading "Graphic Symbol" for the x-axis followed by listing off the numbers, we should instead have an algorithmic way of substituting logical language proscribed by the doenetML that describes the range. In Desmos, this sounds like "Graph paper. X axis visible from negative 5.9 to 5.9. Y axis visible from negative 0.9 to 10.9. Grid lines are visible...." (It ends with how to interact with an audio trace, which we don't have yet.)

-Another example of features that should have descriptive reading text are objects like points, functions, and curves. Currently, these are just read off as "Graphic Symbol". 

Should we make individual issues on all of these graphical elements other than the axes? 

- Labels of graphical objects appear to be ignored by the screen reader

-Finally, the graph navigation is read off as "selectable text" as the last of the features that are read. It is read as "end dash, o (oh), plus"

Here is the sample Desmos activity that I looked at: https://classroom.amplify.com/activity/685c364b05fb0ecc513ebeca?_gl=1*zarf3h*_gcl_au*MjEyODE4NDYwOS4xNzYxNTEwNTYw*_ga*MTcwNjYzOTIyMC4xNzYxNTEwNTYw*_ga_KB37BKPPF6*czE3NjE1MTA1NTkkbzEkZzAkdDE3NjE1MTA1NjIkajU3JGwwJGgw&collections=651ca31cf69ee59aa9e3818a%2C68da94e0a0a96c33d5677a4f%2C68263e6a698c70b9a3b5264e#preview/ce07d03d-609d-43e2-8ad9-6f1bf7f6f9fd

and here is a doenet activity that demonstrates the behavior of a graph in different contexts: https://beta.doenet.org/documentEditor/uXjZSd1dkc4DXpP9MSpqq2/settings?showRequired
